src/core/features/amplitude-factor-feature.h

```diff
@@ -0,0 +1,21 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2023 Intel Corporation. All Rights Reserved.
+
+#pragma once
+
+#include <core/features/feature-interface.h>
```

Comment:
Let's not assume that src/ is part of the include path: I suggest `<src/core/...>`

Reply:
Fixed